torchbenchmark/models/pplbench_beanmachine/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/facebookresearch/pplbench.git#egg=pplbench[ppls]
```

Comment:
Please remove the suffix `#egg=pplbench[ppls]`, looks it is not useful.
Also, please fix the repo version to the latest commit hash `#3767d2d`

Reply:
There is an issue with jsonarparse v4 being addressed in https://github.com/facebookresearch/pplbench/pull/92. They have not made a release with this change yet. So, I was asked to use this specific release

Reply:
I see, thanks for the clarification!